src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1171,7 +1171,7 @@
     <value>Cannot implicitly convert type '{0}' to '{1}'. An explicit conversion exists (are you missing a cast?)</value>
   </data>
   <data name="ERR_PartialMisplaced" xml:space="preserve">
-    <value>The 'partial' modifier can only appear immediately before 'class', 'record', 'struct', 'interface', or a method or property return type.</value>
+    <value>The 'partial' modifier can only appear immediately before 'class', 'record', 'struct', 'interface', 'event', an instance constructor identifier, or a method or property return type.</value>
```

Comment:
Our [learning docs](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/constructors) say: "A constructor is a method with the same name as its type." If you want, we could say "a constructor name" here.